### PR TITLE
Update task 3

### PR DESCRIPTION
**Pull request by daniloczar**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+const { getAllTopics, getAllEndPoints } = require("./controllers/topics.controller");
+const { customErrors, psqlErrors, serverErrors } = require('./errors/errors.js');
+const app = express();
+
+app.get("/api/topics", getAllTopics);
+app.get('/api', getAllEndPoints)
+
+app.use(customErrors);
+app.use(psqlErrors);
+app.use(serverErrors);
```
Comment (nc-john-hutcheon) [inline app.js:11]: Are all of these error handlers being used yet? If not, they should be removed until they are.

File: models/topics.model.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+const db = require("../db/connection.js");
+const fs = require("fs/promises");
+
+exports.selectAllTopics = () => {
+  return db.query("SELECT * FROM topics").then((res) => {
+    return res.rows;
+  });
+};
+
+exports.selectAllEndpoints = () => {
```
Comment (nc-john-hutcheon) [inline models/topics.model.js:10]: We don't actually need a model here, we can just require in the endpoints.json into the controllers file, then send it as a response from there 🙂

**Review by nc-john-hutcheon — COMMENTED** [review]
Hi Danilo, I've just left some comments here, we don't need a model in this instance as we're not making a database query. I'll leave it up to you if you want to refactor 🙂

File: models/topics.model.js (hunk 2 above)
Comment (daniloczar) [reply models/topics.model.js:10]: I ill refactor all now, thank you

**Pull request merged**